docs/custom-matchers/writing-custom-matcher.md

```diff
@@ -15,8 +16,9 @@ const endsWithMatcher = pattern => {
       const wrapRegExp = new RegExp(`(.*)\.${pattern}$`); // In production application pattern should definitely be escaped!
 
       return {
-        unwrap: pattern,
-        wrap: action.type.match(wrapRegExp)[1],
+        id: `ends-with${pattern}`,
+        unwrappedType: pattern,
+        wrap: type => `${action.type.match(wrapRegExp)[1]}.${type}`,
```

Comment:
Maybe it's because I'm reading this out of context, but I have no idea what this is or why we need it.